tests/conftest.py

```diff
@@ -18,6 +18,7 @@
     r".*test_possibly_undefined_checker.*",
     r".*test_redundant_assignment_checker.*",
     r".*test_cfg_generator_z3.*",
+    r".*test_examples.*",
```

Comment:
So some examples use Z3 but others don't. I think it's too restrictive to add all `test_examples` here.

Instead, you can create a separate test name for just for the Z3-related examples and include a regex matching just those in this list.